packages/core/src/render3/i18n/i18n_parse.ts

```diff
@@ -266,15 +295,22 @@ export function i18nAttributesFirstPass(
 /**
  * Generate the OpCodes to update the bindings of a string.
  *
+ * @param updateOpCodes Place where the update opcodes will be stored.
  * @param str The string containing the bindings.
  * @param destinationNode Index of the destination node which will receive the binding.
  * @param attrName Name of the attribute, if the string belongs to an attribute.
  * @param sanitizeFn Sanitization function used to sanitize the string after update, if necessary.
  */
 export function generateBindingUpdateOpCodes(
-    str: string, destinationNode: number, attrName?: string,
-    sanitizeFn: SanitizerFn|null = null): I18nUpdateOpCodes {
-  const updateOpCodes: I18nUpdateOpCodes = [null, null];  // Alloc space for mask and size
+    updateOpCodes: I18nUpdateOpCodes, str: string, destinationNode: number, attrName?: string,
```

Comment:
The `attrName` can not be optional, since the next argument is not optional too, the proposed update is:
```suggestion
    updateOpCodes: I18nUpdateOpCodes, str: string, destinationNode: number, attrName: string|undefined,
```